CMakeLists.txt

```diff
@@ -35,6 +35,8 @@ target_link_libraries (pclbo
     PUBLIC
         PCL::PCL
     PRIVATE
+        pthread
+        Boost::thread
```

Comment:
why is it necessary to link to pthread and boost's thread lib?

Reply:
I'm sorry I was sure my compiler complained about pthread! But you are right it is ok without!